service/glacier/api_client.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 const ServiceID = "Glacier"
+const ServiceAPIVersion = "2012-06-01"
```

Comment:
We probably won't need `ServiceAPIVersion` exported

Reply:
This is generated by smithy-go

Reply:
It was added in another PR to smithy-go. Not sure benefit to user so much but useful for debugging potentially.  Is this an attribute that should be included in the user agent for the client? or does the request already capture that value?